src/components/CompaniesGrid/style.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(330px, 1fr));
+  margin-top: 70px;
+  row-gap: 1.7rem;
+  column-gap: 20px;
+
+  .companyItem {
+    background: #d9d9d9;
```

Comment:
Try to use color variables for consistency - you can add more colors from our Figma design system if they don't exist in `styles/vars.scss` yet